docroot/themes/humsci/humsci_basic/src/scss/utilities/_raised-cards.scss

```diff
@@ -6,6 +6,10 @@
   &--uniform-height .field-hs-row-components .hb-grid__item,
   &--uniform-height .hb-grid .hb-grid__item {
     display: flex;
+
+    & div:only-child {
+      width: 100%;
```

Comment:
This works great! I didn't see any issues although I would consider using `width: inherit` as it would still allow for the width that is set on the parent to trickle down to the card as needed without being as explicit.

Reply:
Also check out the code on lines 22 - 24 for `.hb-raised-cards--uniform-height .hb-card`. With the changes above, we may not need to set the width at 100% here anymore.

Reply:
I think this is valid, when I remove it and test it I do not see any regression. I'll go ahead and remove it and please follow up on your testing as well. Great suggestion!